ui/src/components/connect/LifeGroups.js

```diff
@@ -0,0 +1,439 @@
+import {
+  Box,
+  Button,
+  Center,
+  Flex,
+  Heading,
+  Image,
+  LinkBox,
+  LinkOverlay,
+  SimpleGrid,
+  Spacer,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import { AiOutlineInstagram } from "react-icons/ai";
+import Slider from "react-slick";
+import TestimonyCard from "./TestimonyCard";
+import testimoniesList from "./testimonies.json";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "./_connect.scss";
+
+const LifeGroups = (props) => {
+  const sliderSettings = {
+    adaptiveHeight: true,
+    arrows: false,
+    centerMode: true,
+    dots: false,
+    focusOnSelect: true,
+    infinite: false,
+    slidesPerRow: 1,
+    speed: 500,
+    swipeToSlide: true,
+    variableWidth: true,
+  };
+
+  return (
+    <Box>
+      <Heading
+        as="h2"
+        fontSize={["2.25em", "3em", "4em"]}
+        fontWeight={800}
+        textAlign="center"
+        mb={[0, 4]}
+      >
+        <span style={{ color: "#FD7B7E" }}>L</span>
+        <span style={{ color: "#43B77B" }}>I</span>
+        <span style={{ color: "#7DABFC" }}>F</span>
+        <span style={{ color: "#FEDD64" }}>E</span> GROUPS
+      </Heading>
+      <Box>Life group video</Box>
+      <Flex
+        flexDirection={props.isLargerThan768 ? "row" : "column-reverse"}
+        mb={4}
+      >
+        <Box flex={1}>
+          <Image
+            src={`${process.env.PUBLIC_URL}/images/connect/LG-map.png`}
+            mr={[0, 6]}
+          />
+        </Box>
+        <Stack flex={1} textAlign={["center", "left"]} pb={[6, 0]}>
+          <Text fontSize={["md", "xl"]} mb={4}>
+            At HMCC, we believe in the power of community and the fullness of
+            life that it offers -- which is why we believe in{" "}
+            <span style={{ fontWeight: "bold" }}>LIFE Groups.</span>
+            <br />
+            <br />
+            LIFE Group is more than just a weekly Bible study group. It is about{" "}
+            <span style={{ fontWeight: "bold" }}>Loving</span> one another,{" "}
+            <span style={{ fontWeight: "bold" }}>Investing</span> in the
+            community, growing in our{" "}
+            <span style={{ fontWeight: "bold" }}>Faith</span>, and{" "}
+            <span style={{ fontWeight: "bold" }}>Enjoying</span> life together
+            as a family living out the Gospel.
+          </Text>
+          <Center>
+            <LinkBox
```

Comment:
why not make a component for this instead of having tons of these pretty massive boxes?

Reply:
yeah u right